data/profiles/artemis.profile

```diff
@@ -0,0 +1,137 @@
+    },
+    {
+      "documentation": "Retargets the camera on Earth",
+      "gui_path": "/Solar System/Earth",
+      "identifier": "profile.focus.earth",
+      "is_local": false,
+      "name": "Focus on Earth",
+      "script": "openspace.setPropertyValueSingle('NavigationHandler.OrbitalNavigator.Aim', '');openspace.setPropertyValueSingle('NavigationHandler.OrbitalNavigator.Anchor', 'Earth')openspace.setPropertyValueSingle('NavigationHandler.OrbitalNavigator.RetargetAnchor', nil);"
+    }
+  ],
+  "assets": [
+    "base",
+    "events/toggle_sun",
+    "scene/solarsystem/planets/earth/earth",
+    "scene/solarsystem/planets/earth/satellites/satellites",
+    "scene/solarsystem/missions/artemis/artemis"
+  ],
+  "camera": {
+    "altitude": 17000000.0,
+    "anchor": "Earth",
+    "latitude": 58.5877,
+    "longitude": 16.1924,
+    "type": "goToGeo"
+  },
+  "delta_times": [
+    1.0,
+    5.0,
+    30.0,
+    60.0,
+    300.0,
+    1800.0,
+    3600.0,
+    43200.0,
+    86400.0,
+    604800.0,
+    1209600.0,
+    2592000.0,
+    5184000.0,
+    7776000.0,
+    15552000.0,
+    31536000.0,
+    63072000.0,
+    157680000.0,
+    315360000.0,
+    630720000.0
+  ],
+  "keybindings": [
+    {
+      "action": "profile.toggle.satellite",
+      "key": "S"
+    },
+    {
+      "action": "profile.focus.iss",
+      "key": "I"
+    },
+    {
+      "action": "profile.focus.earth",
+      "key": "HOME"
+    }
+  ],
+  "mark_nodes": [
+    "ArtemisModel",
+    "Earth",
+    "Moon",
+    "Sun"
+  ],
+  "meta": {
+    "author": "OpenSpace Team",
+    "description": "Artemis Profile. Adds the Orion capsule (Artemis-1) model with an estimated trajectery",
+    "license": "MIT License",
+    "name": "Artemis",
+    "url": "https://www.openspaceproject.com",
+    "version": "1.0"
+  },
+  "properties": [
+    {
+      "name": "{earth_satellites}.Renderable.Enabled",
+      "type": "setPropertyValue",
+      "value": "false"
+    },
+    {
+      "name": "Scene.MoonTrail.Renderable.Appearance.Color",
+      "type": "setPropertyValueSingle",
+      "value": "{0.5, 0.5, 0.5}"
+    },
+    {
+      "name": "Scene.MoonTrail.Renderable.Appearance.Fade",
+      "type": "setPropertyValueSingle",
+      "value": "4.0"
+    },
+    {
+      "name": "Scene.Earth.Renderable.Layers.ColorLayers.ESRI_NOAA20_Combo.Enabled",
+      "type": "setPropertyValueSingle",
+      "value": "true"
+    },
+    {
+      "name": "Scene.Earth.Renderable.Layers.ColorLayers.ESRI_VIIRS_Combo.Enabled",
+      "type": "setPropertyValueSingle",
+      "value": "false"
+    },
+    {
+      "name": "Scene.ISSModel.Renderable.Enabled",
+      "type": "setPropertyValueSingle",
+      "value": "false"
+    },
+    {
+      "name": "Scene.ISS_trail.Renderable.Enabled",
+      "type": "setPropertyValueSingle",
+      "value": "false"
+    }
+  ],
+  "time": {
+    "type": "relative",
+    "value": "-1d"
```

Comment:
How about setting the start time of the profile to 2022 DEC 05 15:30:00 since it looks nice?
![image](https://user-images.githubusercontent.com/4921724/231573285-199a350a-7974-43a8-a323-6a473a798968.png)

Could also have the start position at Artemis, too